README.md

```diff
@@ -63,12 +63,24 @@ maxExtLength | number | `4_294_967_295` (UINT32_MAX)
 
 You can use `max${Type}Length` to limit the length of each type decoded.
 
-### `decodeAsync(stream: AsyncIterable<ArrayLike<number> | Uint8Array>, options?: DecodeAsyncOptions): Promise<unknown>`
+### `decodeAsync(stream: AsyncIterable<Uint8Array | ArrayLike<number>> | ReadableStream<Uint8Array | ArrayLike<number>>, options?: DecodeAsyncOptions): Promise<unknown>`
 
 It decodes `stream` in an async iterable of byte arrays and returns decoded data as `uknown` wrapped in `Promise`. This function works asyncronously.
 
 Note that `decodeAsync()` acceps the same options as `decode()`.
 
+### `decodeArrayStream(stream: AsyncIterable<Uint8Array | ArrayLike<number>> | ReadableStream<Uint8Array | ArrayLike<number>>, options?: DecodeAsyncOptions): AsyncIterable<unknown>`
+
+It is alike to `decodeAsync()`, but only accepts an array of items as `stream`, and emits the decoded item one-by-one.
+
+It throws errors when the input is not an array.
```

Comment:
Better to add example scripts (and add it to test/readme.test.ts) in a future.

Reply:
yeah, I think we need code example for it